ext/mbstring/php_unicode.c

```diff
@@ -238,6 +238,41 @@ static uint32_t *emit_special_casing_sequence(uint32_t w, uint32_t *out)
 	return out;
 }
 
+/* Used when determining whether special casing rules should be applied to Greek letter sigma */
+static bool scan_ahead_for_cased_letter(unsigned char *in, size_t in_len, unsigned int state, const mbfl_encoding *encoding)
+{
+	uint32_t wchar_buf[64];
+
+	while (in_len) {
+		size_t out_len = encoding->to_wchar(&in, &in_len, wchar_buf, 64, &state);
+		ZEND_ASSERT(out_len <= 64);
+		for (int i = 0; i < out_len; i++) {
+			uint32_t w = wchar_buf[i];
+			if (php_unicode_is_cased(w))
+				return true;
+			if (!php_unicode_is_case_ignorable(w))
+				return false;
```

Comment:
Nit: coding style, we "mandate" braces even for such simple if statements (yes yes it's not consistently applied but at least let us do this for new code)
```suggestion
			if (php_unicode_is_cased(w)) {
				return true;
			}
			if (!php_unicode_is_case_ignorable(w)) {
				return false;
			}
```

Reply:
True. Thanks for the reminder. Fixed.